docs/stdlib/set.rst

```diff
@@ -140,15 +140,17 @@ Sets
 
 .. eql:operator:: union: set of anytype union set of anytype -> set of anytype
 
-    Merge two sets.
+    Merges two sets of :eql:type:`anytype` together as one.
 
-    Since EdgeDB sets are formally multisets, ``union`` is a *multiset sum*,
-    so effectively it merges two multisets keeping all of their members.
+    Since EdgeDB sets are formally multisets, :eql:op:`union` is a *multiset
+    sum*, so effectively it merges two multisets while keeping all of their
+    members.
```

Comment:
```suggestion
    Since EdgeDB sets are formally multisets, :eql:op:`union` is a *multiset
    sum*, effectively merging two multisets while keeping all of their members.
```